lib/puppet_litmus/rake_helper.rb

```diff
@@ -258,7 +258,7 @@ def install_module(inventory_hash, target_node_name, module_tar, module_reposito
                       target_node_name
                     end
     run_local_command("bundle exec bolt file upload \"#{module_tar}\" /tmp/#{File.basename(module_tar)} --nodes #{target_string} --inventoryfile inventory.yaml")
-    install_module_command = "puppet module install --module_repository #{module_repository} /tmp/#{File.basename(module_tar)}"
+    install_module_command = "puppet module install --force --module_repository #{module_repository} /tmp/#{File.basename(module_tar)}"
```

Comment:
with --force on will it install dependencies ?

Reply:
> Installs the module regardless of dependency tree, checksum changes, or whether the module is already installed. By default, installs the module in the default modulepath, even if the module is already installed in another directory. Does not install dependencies.

FFS!

Reply:
changed it to instead uninstall first. That should solve this.